utilities/titandb/titan_db_test.cc

```diff
@@ -139,6 +142,64 @@ class TitanDBTest : public testing::Test {
     }
   }
 
+  void VerifyBlob(const std::map<std::string, std::string>& data, bool compacted = true ) {
+    const TitanSnapshot* snapshot = dynamic_cast<const TitanSnapshot*>(
+        db_->GetSnapshot());
+    ASSERT_TRUE(snapshot != nullptr);
+    
+    auto* version(snapshot->current());
+    ASSERT_TRUE(version != nullptr);
+    version->Ref();
```

Comment:
Wrap the above logic in a `GetBlobStorage` helper method? Since `TitanSnapshot` and `Version` is going to be removed in #83, let's make it easier to resolve merge conflict later :)